app/migrations/Version20191017140848.php

```diff
@@ -40,7 +40,7 @@ public function up(Schema $schema): void
         }
 
         if (!$smsStatsTable->hasColumn('details')) {
-            $this->addSql("ALTER TABLE {$this->prefix}sms_message_stats ADD details LONGTEXT NOT NULL COMMENT '(DC2Type:json_array)';");
+            $this->addSql("ALTER TABLE {$this->prefix}sms_message_stats ADD details LONGTEXT NOT NULL COMMENT '(DC2Type:json)';");
```

Comment:
This change is based on the [comment](https://github.com/doctrine/orm/issues/9078#issuecomment-951803233) 

and the [code sample here ](https://github.com/doctrine/orm/issues/9078#issuecomment-1141810517)

I think this approach of editing the migration is wrong, I think we need a new migration file to remove the comment from all the columns as in the sample above, but I would love to get your thoughts

Reply:
@mabumusa1 shouldn't we pull [this](https://github.com/doctrine/orm/issues/9078#issuecomment-1141810517) migration for existing Mautic instances too?

Reply:
@escopecz yes, plan to do one migration to make all the changes once. I think the new migration should be added to branch `4.4` to make the upgrade process smooth.